src/NHibernate/Action/IExecutable.cs

```diff
@@ -36,6 +36,11 @@ namespace NHibernate.Action
 	//6.0 TODO: Consider refactoring towards using an abstract class
 	public partial interface IExecutable
 	{
+		/// <summary>
+		/// What UpdateTimestamps Spaces are affected any actions
+		/// </summary>
+		string[] UpdateTimestampsSpaces { get; }
```

Comment:
This is a breaking change, which you can avoid by adding an extension method instead. Maybe also rename it to `QueryCacheSpaces` for easier understanding.

Reply:
QueryCacheSpaces ok I also created new two interface ICacheableExecutable and ICacheableEntityPersister that will moved into original interfaces in nh 6.0